pkg/reconciler/layer3.go

```diff
@@ -283,7 +283,7 @@ func (r *reconcile) cleanupTaasNetlink(existing []nl.TaasInformation, intended [
 	for _, cfg := range existing {
 		stillExists := false
 		for i := range intended {
-			if intended[i].IsTaaS && intended[i].Name == cfg.Name && intended[i].VNI == cfg.Table {
+			if intended[i].Name == cfg.Name && intended[i].VNI == cfg.Table {
```

Comment:
and this `cfg.Table == intended[i].VNI works` is there not a limit on table ids?

Reply:
I am using VNI as the field for tableID as well, limit is 2^32-1, it's a 32bit uint

Reply:
ok. then it fits :)